package.json

```diff
@@ -2,7 +2,7 @@
     "name": "zig",
     "displayName": "Zig",
     "description": "Language support for the Zig programming language",
-    "version": "0.2.2",
+    "version": "0.2.3",
```

Comment:
Can you remove this, will bump the version prior to doing a release.

Reply:
Done https://github.com/ziglang/vscode-zig/pull/20/commits/9c54260bbf44831072f7052f48b11d1ee2baa651